sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -506,7 +506,13 @@ def __init__(self, options, proto_pipeline):
     # Labels.
     if self.google_cloud_options.labels:
       self.proto.labels = dataflow.Job.LabelsValue()
-      for label in self.google_cloud_options.labels:
+      labels = self.google_cloud_options.labels
+      if len(labels) == 1:
+        # labels can be passed as cmd args
+        # --labels=name=wrench,age=32,job=dataflow and this is parsed by
+        # argparse as 'labels' : ["name=wrench,age=32,job=dataflow"]
+        labels = labels[0].split(',')
```

Comment:
I think we should prefer the json format currently specified in https://cloud.google.com/dataflow/docs/reference/pipeline-options#python rather than just doing comma separated since that is consistent with how Java handles this. Whenever we can fairly easily remain consistent, we should do so.

Reply:
That also allows us to be permissive with special characters (e.g. if `,` is part of the label)